data/tlg0007/tlg082/tlg0007.tlg082.perseus-eng3.xml

```diff
@@ -0,0 +1,4317 @@
+                  </note>
+					          </p>
+            </div>
+            <div type="textpart" subtype="section" n="65">
+               <p rend="indent">Being asked how one might most surely have a good name among men, he said,
+							<q type="unspecified">If one say what is best and do what is most honourable.</q>
+					          </p>
+            </div>
+            <div type="textpart" subtype="section" n="66">
+               <p rend="indent">He used to say that a general ought to be possessed of boldness towards the
+						enemy and kind ness towards the men under him. <note place="unspecified" anchored="true">Stobaeus, <title rend="italic" xml:lang="lat">Florilegium</title>, liv. 49, adds, <q type="unspecified">and reasoning power
+							to meet crises.</q>
+                  </note>
+               </p>
+            </div>
+            <div type="textpart" subtype="section" n="67">
+               <p rend="indent">When someone desired to know what boys ought to learn, he said, <q type="unspecified">That
+							which they will use when they become men.</q>
+						            <note place="unspecified" anchored="true">Leotychidas (224 D, <foreign xml:lang="lat">infra</foreign>) and Aristippus
+							(diogenes Laertius, ii. 80) expressed the same opinion, which has been
+							repeated <foreign xml:lang="lat">ad nauseam</foreign> by professors of
+							paedagogy.</note>
+					          </p>
+            </div>
+            <div type="textpart" subtype="section" n="68">
+               <p rend="indent">Once when he was sitting as judge the accuser spoke well and the defendant
+						poorly, merely repeating in answer to each point, <q type="unspecified">Agesilaus, a king
+							must uphold the laws;</q> whereat Agesilaus said, <q type="unspecified">And if
+							somebody had broken into your house, and if somebody had robbed you of
+							your coat, should you expect that the builder of the house or the maker
+							of the coat would come to your assistance?</q>
+					          </p>
+            </div>
+            <div type="textpart" subtype="section" n="69">
+               <p rend="indent">After the peace was made, <note place="unspecified" anchored="true">The peace of Antalcidas, 387 B.C. See <title rend="italic" xml:lang="lat">Moralia</title>, 213 A-B, <foreign xml:lang="lat">supra</foreign>.
+                  </note> a letter from the king of Persia was brought to him,
+						of which the <pb xml:id="v.3.p.277"/> Persian with Gallias the Spartan was the
+						bearer, in regard to hospitality and friendship; but Agesilaus would not
+						receive it, bidding the man to take back word to the king that there was no
+						need to send letters to him personally; that if the king showed himself to
+						be a friend to Sparta and well disposed towards Greece, he himself, to the
+						very best of his power, wou
```

Comment:
There is a typo in the print here, actually. This should be section 5. (Hook test returned a duplicate reference at 38.4).